### PR TITLE
DOCSP-30417 Project Migration

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Project drawer migration.

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30417/projects/create-project-live-connection/

JIRA:  https://jira.mongodb.org/browse/DOCSP-30417

BUILD: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=648c80e6259a8493e5f82a2b

### CODE REVIEW

**Review by sarah-olson-mongodb — COMMENTED** [review]
Really great work @jocelyn-mendez1! I've left a bunch of minor edits and a few markup change requests. Happy to take another look this afternoon!

File: source/projects/create-project-live-connection.txt [hunk 1]
```diff
@@ -3,3 +3,153 @@
...
+
+      Connection and SSL details depend on the database type you are connecting to. 
+      In addition to the generic connection properties listed above, you may also 
+      need to select an ``Identifier`` for Oracle and ``Authentication`` for SQL Server.
+
+.. note::
+
+   If you want to specify the JDBC URI manually, click the :guilabel:`Enter URI manually` 
+   toggle switch on the :guilabel:`Connect to the database screen`. For details see, :ref:`Relational Database Connection Strings <rm-relational-database-connection-strings>`.
+
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-live-connection.txt:94]: Done.

File: source/projects/create-project-live-connection.txt [hunk 2]
```diff
@@ -3,3 +3,153 @@
...
+      avoid reentering your password in the project.
+   
+   g. Click the :guilabel:`SSL` toggle switch to enable or disable SSL and select an SSL mode. 
+
+   h. Click :guilabel:`Connect`.
+
+      Connection and SSL details depend on the database type you are connecting to. 
+      In addition to the generic connection properties listed above, you may also 
+      need to select an ``Identifier`` for Oracle and ``Authentication`` for SQL Server.
+
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-live-connection.txt:88]: Done.

File: source/projects/create-project-live-connection.txt [hunk 3]
```diff
@@ -3,3 +3,153 @@
...
+   The following table explains the different ways you can select tables to migrate: 
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - Target
+        - Action
+
+      * - All tables within a database.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-live-connection.txt]: Done.

File: source/projects/create-project-live-connection.txt [hunk 4]
```diff
@@ -3,3 +3,153 @@
...
+        - Click the check mark for the target database.
+
+      * -  All tables within a schema.
+        - Expand the target database and click the check mark for the target schema.
+
+      * - Specific tables within a schema.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-live-connection.txt]: Done.

File: source/projects/create-project-live-connection.txt [hunk 5]
```diff
@@ -3,3 +3,153 @@
...
+   - :guilabel:`Override with Global Casing`: Override the original table name
+     with a global casing convention.
+
+     - ``camelCase``
+     - ``TitleCase``
+     - ``kebab-case``
+     - ``snake_case``
+     - ``UPPER_SNAKE_CASE``
+
+#. Choose an :guilabel:`Initial mappings` option for your MongoDB schema:
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-live-connection.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 6]
```diff
@@ -1,3 +1,180 @@
...
+   from these database types: 
+
+   - MySQL
+   - Oracle
+   - PostgreSQL
+   - SQL Server
+
+#. Choose or drag and drop a DDL statement file and click :guilabel:`Import`.
+
+#. From the :guilabel:`Select tables` screen, indicate the tables you want to migrate
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 7]
```diff
@@ -1,3 +1,180 @@
...
+   The following table explains the different ways you can select tables to migrate: 
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - Target
+        - Action
+
+      * - All tables within a database.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 8]
```diff
@@ -1,3 +1,180 @@
...
+        - Click the check mark for the target database.
+
+      * -  All tables within a schema.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 9]
```diff
@@ -1,3 +1,180 @@
...
+        - Expand the target database and click the check mark for the target schema.
+
+      * - Specific tables within a schema.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 10]
```diff
@@ -1,3 +1,180 @@
...
+        - Expand the target database and schema. Select the target tables individually.
+    
+      * - Specific table names.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-loading-schema-files.txt [hunk 11]
```diff
@@ -1,3 +1,180 @@
...
+   - :guilabel:`Override with Global Casing`: Override the original table name
+     with a global casing convention.
+
+     - ``camelCase``
+     - ``TitleCase``
+     - ``kebab-case``
+     - ``snake_case``
+     - ``UPPER_SNAKE_CASE``
+
+#. Choose an :guilabel:`Initial mappings` option for your MongoDB schema:
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-loading-schema-files.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 12]
```diff
@@ -1,3 +1,138 @@
...
+Before you Begin
+----------------
+
+Before you can run sync jobs from the sample schema you must either:
+
+1. Download `Docker <https://www.docker.com/>`__ to pull and host a provided 
+   PostgreSQL database container. This container comes preloaded with the sample 
+   schema presented in Relational Migrator.
+
+- Download and execute the `Northwind.sql <https://github.com/pthom/northwind_psql/blob/master/northwind.sql>`__ database setup script on an existing PostgreSQL instance. This
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 13]
```diff
@@ -1,3 +1,138 @@
...
+
+   The connection credentials for this database instance are:
+
+   - Username: ``postgres``
+
+   - Password: ``postgres``
+
+   - Port: ``5432``
+
+   .. warning
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 14]
```diff
@@ -1,3 +1,138 @@
...
+     The PostgreSQL database is for demo uses only and should not be used for 
+     production purposes.
+
+#. In Relational Migrator, click :guilabel:`New Project` at the top-right of the 
+   Relational Migrator home screen.
+
+#. Click the :guilabel:`Create sample` button.
+
+#. From the :guilabel:`Select tables` screen, indicate the tables you want to 
+   migrate and click :guilabel:`Next`.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 15]
```diff
@@ -1,3 +1,138 @@
...
+   The following table explains the different ways you can select tables to migrate: 
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - Target
+        - Action
+
+      * - All tables within a database.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 16]
```diff
@@ -1,3 +1,138 @@
...
+        - Click the check mark for the target database.
+
+      * -  All tables within a schema.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 17]
```diff
@@ -1,3 +1,138 @@
...
+        - Expand the target database and click the check mark for the target schema.
+
+      * - Specific tables within a schema.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 18]
```diff
@@ -1,3 +1,138 @@
...
+        - Expand the target database and schema. Select the target tables individually.
+    
+      * - Specific table names.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/create-project-sample-schema.txt [hunk 19]
```diff
@@ -1,3 +1,138 @@
...
+   - :guilabel:`Override with Global Casing`: Override the original table name
+     with a global casing convention.
+
+     - ``camelCase``
+     - ``TitleCase``
+     - ``kebab-case``
+     - ``snake_case``
+     - ``UPPER_SNAKE_CASE``
+
+#. Choose an :guilabel:`Initial mappings` option for your MongoDB schema:
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/create-project-sample-schema.txt]: Done.

File: source/projects/import-project.txt [hunk 20]
```diff
@@ -3,3 +3,52 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to import projects. Import a project when you 
+want to collaborate on a shared project. You can also duplicate a project by 
+:ref:`exporting <export-project>` the project and then importing it. 
+
+About This Task
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/projects/import-project.txt]: Done.

**Comment by jocelyn-mendez1** [comment]
Hi @sarah-olson-mongodb, thanks for the feedback. I have made the updates so let me know of any other possible changes. Thanks!

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks very much @jocelyn-mendez1. Great work!

**Pull request merged**